test/models/post_test.rb

```diff
@@ -1,7 +1,71 @@
 require 'test_helper'
 
```

Comment:
- [ ] It would be better if we add our unit spec testing. I have tried running the test with `rspec spec` , `rspec` but here is the results below
![Screenshot from 2023-03-30 15-03-32](https://user-images.githubusercontent.com/46388113/228879126-ff9c743d-ecfd-4af9-bc42-67989151bdbe.png)
AS you can see in the requirements below
![Screenshot from 2023-03-30 15-01-24](https://user-images.githubusercontent.com/46388113/228879269-1994752a-5254-460e-a702-d6a23caac85d.png)
This is why i reached out to you on slack so we can clarify it before proceeding but unfortunately you did not respond

Reply:
Hello @addod19, 

Thans for your feedback. I have used MiniTest framework. You can use `rails test` to run test cases. 

Thanks again.